### DISCUSSION TITLE
Chinese LLaMA and Alpaca available, seamlessly work with llama.cpp

### DISCUSSION THREAD

**Discussion by ymcui** [ggml-org/llama.cpp#567] [General]
Greetings,

Thank you for providing such a wonderful tool for LLM deployment on CPU. 
We released Chinese LLaMA and Alpaca model (with Chinese vocabulary extension!), trained with Alpaca-LoRA.
We managed to successfully serve our quantized model with llama.cpp.

Give it a try at: https://github.com/ymcui/Chinese-LLaMA-Alpaca/blob/main/README_EN.md

![](https://github.com/ymcui/Chinese-LLaMA-Alpaca/raw/main/pics/screencast.gif)

**Comment by linouxis9** [comment]
That's great! How does your model compare to https://github.com/THUDM/ChatGLM-6B?
